.circleci/config.yml

```diff
@@ -55,3 +55,11 @@ jobs:
       - run:
           name: Project Tests
           command: yarn test:projects:cra-ts
+      - run:
+          name: Publish npm package
+          command: |
+            if [ ! -z ${npm_TOKEN+x} ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
Check for `npm_TOKEN` can be used to disable publishing without any commit if something goes wrong.
It is only temporary and will be removed once the step proofs it is stable.

Reply:
couple of comments:
- why `+x` trick is necessary for parameter expansion in this condition? seems that `if [ -n "${npm_TOKEN}" ]` would do the same test (and is easier to read)
- would suggest to use double quotes for wrapping this expression

Reply:
I am used to add `+x` to distinguish between unset and empty string. Although it is not necessary here, I consider it a defensive programming technique.
Will switch the condition to `-n` and add quotes.